cypress/features/regression/test/draggable.feature

```diff
@@ -0,0 +1,28 @@
+Feature: Draggable Component
+  I want to change Draggable component
+
+  Background: Open Draggable Component in iframe
+    Given I open "test-draggable" component page basic in iframe
```

Comment:
can you use here step `I open basic Test {string} component page` which we have to open new `basic` stories ?

Reply:
good point, fixed